src/Identity/UI/src/UIFrameworkAttribute.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Identity.UI
+{
+    /// <summary>
+    /// The UIFramework Identity UI will use on the application.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Assembly, Inherited = false, AllowMultiple = false)]
+    public class UIFrameworkAttribute : Attribute
```

Comment:
Does this need to be public? This should also be declared sealed.

Reply:
I'm trying to get rid of this attribute too

Reply:
Couldn't remove it.

It needs to be public because it gets stamped into the app assembly. I marked it sealed, though